ReactAndroid/src/main/java/com/facebook/react/common/ViewHelperMethods.java

```diff
@@ -0,0 +1,22 @@
+package com.facebook.react.common;
+
+import android.view.View;
+
+/**
+ * Various helper methods for {@link View}
+ */
+public class ViewHelperMethods {
```

Comment:
`*Util` is more common.  Renaming to that.  I couldn't find an existing class to put this in.